cumulusci/tasks/salesforce/custom_settings_wait.py

```diff
@@ -73,22 +73,25 @@ def _poll_action(self):
         except SalesforceError as e:
             message = e.content[0]["message"]
             if "No such column 'SetupOwnerId'" in message:
-                message = "Only Hierarchical Custom Settings objects are supported."
+                message = "Only Custom Settings objects are supported."
```

Comment:
@force2b So is this also aiming to support non-hierarchical custom settings? If so the description for the `object` option needs to be tweaked above too.

Reply:
Good point. I think I accidentally removed that line. Will put it back.